packages/cli/src/config/schema.ts

```diff
@@ -907,7 +900,7 @@ export const schema = {
 			env: 'N8N_DEFAULT_BINARY_DATA_MODE',
 			doc: 'Storage mode for binary data',
 		},
-		localStoragePath: {
```

Comment:
Will do. I didn't know the config file syntax was similar to `schema.ts`. It wish it were just a list of env vars.

Reply:
Renamed back to `localStoragePath` at [04f4b83](https://github.com/n8n-io/n8n/pull/7164/commits/04f4b838840ea3e40c261a3d38d618d55ade3be8), since I'd rather not have to bother users for this.